nodejs/week2/routers/searchId.js

```diff
@@ -0,0 +1,20 @@
+import express from "express";
+const router = express.Router();
+//import documents from "../documents.json" assert { type: "json" };
```

Comment:
Let's get into the habit of not leaving commented code in the commit

Reply:
Sure! Thanks for your review